README.md

```diff
@@ -3,9 +3,9 @@
 [![npm version](https://badge.fury.io/js/ember-template-lint.svg)](https://badge.fury.io/js/ember-template-lint)
 [![Build Status](https://github.com/ember-template-lint/ember-template-lint/workflows/CI/badge.svg)](https://github.com/ember-template-lint/ember-template-lint/actions?query=workflow%3ACI)
 
-ember-template-lint will lint your handlebars template and return error results.
+Ember Template Lint is a library that will lint your handlebars template and return error results.
```

Comment:
Let's keep the name of the package here rather than the other. Other than that, looks great!

```suggestion
`ember-template-lint` is a library that will lint your handlebars template and return error results.
```